source/FreeRTOS_UDP_IPv6.c

```diff
@@ -425,20 +428,17 @@ BaseType_t xProcessReceivedUDPPacket_IPv6( NetworkBufferDescriptor_t * pxNetwork
     {
         if( pxSocket != NULL )
         {
-            if( *ipLOCAL_IP_ADDRESS_POINTER != 0U )
-            {
-                /* When refreshing the ARP/ND cache with received UDP packets we must be
-                 * careful;  hundreds of broadcast messages may pass and if we're not
-                 * handling them, no use to fill the ARP cache with those IP addresses. */
-                pxUDPPacket_IPv6 = ( ( UDPPacket_IPv6_t * ) pxNetworkBuffer->pucEthernetBuffer );
-                vNDRefreshCacheEntry( &( pxUDPPacket_IPv6->xEthernetHeader.xSourceAddress ), &( pxUDPPacket_IPv6->xIPHeader.xSourceAddress ),
-                                      pxNetworkBuffer->pxEndPoint );
-            }
-            else
+            if( xCheckRequiresARPResolution( pxNetworkBuffer ) == pdTRUE )
```

Comment:
This one is needed for "ping winsim".
Issue is pxUDPPacket_IPv6 not being initialized.